src/Validator/index.js

```diff
@@ -122,3 +122,11 @@ export const validateDarkTheme = (parameter?: any): boolean => {
 
   return window.matchMedia('(prefers-color-scheme: dark)').matches
 }
+
+export const validateCookieName = (parameter?: any): string => {
```

Comment:
Can you place this function just before the `validateDarkTheme()` function please ?